templates/vue/src/components/tyde/TydeIcon.vue

```diff
@@ -31,19 +40,49 @@ export default {
       type: Boolean,
       required: true,
     },
+    placement: {
+      type: String,
+      required: true,
+    },
   },
   data() {
     return {
       url: "",
+      windowWidth: window.innerWidth,
     }
   },
+  computed: {
+    title() {
+      let title = " "
+      switch (this.icon) {
+        case "default":
+          title = "Learning Program"
+          break
+        case "profile":
+          title = "Profile"
+          break
+        case "goals":
+          title = "Goals"
+          break
+        case "cos":
+          title = "Circle of Support"
+          break
+      }
+
+      return title
+    },
+  },
   watch: {
     selected() {
       this.selectIcon()
```

Comment:
In the `selectIcon()` method, can simplify each case to a ternary expression like `selectedIcon = this.selected ? tyde : tydeUnselected`

Reply:
Nice catch